WS2012R2/lisa/remote-scripts/ica/NUMA_off_kernel.sh

```diff
@@ -0,0 +1,158 @@
+#!/bin/bash
+
+########################################################################
+#
+# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+#
+# All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the ""License"");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+# ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+# PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+#
+########################################################################
+
+########################################################################
+#
+# Description:
+#	This script appends "numa=off" to kernel boot params
+#   RHEL-6.x and 7.x are supported, and both generation 1 and 2
+#   are supported as well.
+#
+################################################################
+
+ICA_TESTRUNNING="TestRunning"      # The test is running
+ICA_TESTCOMPLETED="TestCompleted"  # The test completed successfully
+ICA_TESTABORTED="TestAborted"      # Error during setup of test
+ICA_TESTFAILED="TestFailed"        # Error during execution of test
+
+UpdateTestState() {
+    echo $1 > $HOME/state.txt
+}
+
+UpdateSummary() {
+	# To add the timestamp to the log file
+    echo `date "+%a %b %d %T %Y"` : ${1} >> ~/summary.log
+}
+
+cd ~
+if [ -e ~/summary.log ]; then
+    rm -rf ~/summary.log
+fi
+
+UpdateTestState $ICA_TESTRUNNING
+
+if [ -e $HOME/constants.sh ]; then
+    . $HOME/constants.sh
+else
+    echo "ERROR: Unable to source the constants file!"
+    UpdateSummary "ERROR: Unable to source the constants file!"
+    UpdateTestState $ICA_TESTABORTED
+    exit 1
+fi
+
+# Convert eol
+dos2unix utils.sh
+
+# Source utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    UpdateSummary "Error: unable to source utils.sh!"
+    UpdateTestState $ICA_TESTABORTED
+    exit 1
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Get distro
+GetDistro
+
+#
+# Add "numa=off" into kernel boot parameter
+#
+case $DISTRO in
```

Comment:
We try to design test cases to handle different distributions, unless it is really a distro specific feature.
Would you be able to add a distro case at least for ubuntu and sles as well?
Otherwise we can make that addition, however we prefer to cover at least these 3 main distributions.
Thank you!